### PR TITLE
Master To Formplayer Merge

### PR DESCRIPTION
**Pull request by shubham1g5**

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]

File: src/cli/java/org/commcare/util/cli/ApplicationHost.java [hunk 1]
```diff
@@ -109,15 +112,63 @@ public void setRestoreToDemoUser() {
...
+            endpointArgs = new String[0];
+        }
+
+        mSession.clearAllState();
+        mSession.clearVolatiles();
+
+        EvaluationContext evalContext = mSession.getEvaluationContext();
+        try {
+            Endpoint.populateEndpointArgumentsToEvaluationContext(endpoint, new ArrayList<String>(Arrays.asList(endpointArgs)), evalContext);
+        } catch (Endpoint.InvalidEndpointArgumentsException e) {
```
Comment (orangejenny) [inline src/cli/java/org/commcare/util/cli/ApplicationHost.java:134]: @shubham1g5 Are we really going to have a 100-char limit? Even in python we have 118, and Java is more verbose.

**Comment by orangejenny** [comment]
(did not review closely since it's a merge)

File: src/cli/java/org/commcare/util/cli/ApplicationHost.java (hunk 1 above)
Comment (shubham1g5) [reply src/cli/java/org/commcare/util/cli/ApplicationHost.java:134]: I am happy to change it to 120. 100 is a standard at Google where I took the styles files from.

Comment (orangejenny) [reply src/cli/java/org/commcare/util/cli/ApplicationHost.java:134]: It'll affect you more than me, but yeah, bumping it up a little sounds good 🙂

**Comment by orangejenny** [comment]
@shubham1g5 Are you having QA do a formplayer regression for this?

**Comment by shubham1g5** [comment]
Yes I do plan to have it go through QA.

**Comment by shubham1g5** [comment]
QA has passed on this. Mergin.

**Pull request merged**